packages/blade/src/components/Spinner/BaseSpinner/BaseSpinner.tsx

```diff
@@ -6,9 +6,22 @@ import type { Theme } from '~components/BladeProvider';
 import { useTheme } from '~components/BladeProvider';
 import { getIn, makeAccessible, makeSize } from '~utils';
 import type { ColorContrastTypes, Feedback } from '~tokens/theme/theme';
+import Box from '~components/Box';
+import { Text } from '~components/Typography';
 
 type BaseSpinnerProps = {
   intent?: Feedback;
+  /**
+   * Sets the label of the spinner.
+   *
+   * @default 'right'
+   */
+  label?: string;
```

Comment:
Now that we've both label and a11y label. What if user provides different values for both 🤔

Reply:
That's a valid case. The way this works is, we set a11y label's value as the prop a11yLabel, if it doesn't exist then we set it as the value of the prop label else we fall back to "Loading"